library/src/main/kotlin/it/krzeminski/githubactions/actions/madhead/ReadJavaProperties.kt

```diff
@@ -0,0 +1,21 @@
+package it.krzeminski.githubactions.actions.madhead
+
+import it.krzeminski.githubactions.actions.Action
+
+data class ReadJavaProperties(
+    val file: String,
+    val property: String? = null,
+    val all: Boolean = false,
```

Comment:
I'm trying avoiding putting defaults in the wrappers. How about having this here:
```suggestion
    val all: Boolean? = null,
```
meaning _"I don't really care what the default is, I'm letting the action decide"_, and in `toYamlArguments()` something like:

```kotlin
all?.let { result["all"] = it.toString() }
```

Makes sense?

Reply:
Done